packaging/rpm-ostree.spec.in

```diff
@@ -135,17 +135,14 @@ The %{name}-devel package includes the header files for %{name}-libs.
 %autosetup -Sgit -n %{name}-%{version}
 
 %build
-# gobject introspection does not work with LTO.  There is an effort to fix this
-# in the appropriate project upstreams, so hopefully LTO can be enabled someday
-# Disable LTO.
-%define _lto_cflags %{nil}
```

Comment:
To confirm, you tested this on el8 too I presume?

Reply:
Also, we could work around this by skipping LTO in just the g-i part - in other words rather than injecting global LTO flags into our build, we change to do it by default (skipping g-i) and maybe have a `--disable-lto` if really necessary.

Reply:
This is confirmed to work on el9 so far. I haven't yet got to a full successful build on el8 due other blockers / missing dependencies.
Let's keep this hanging a bit more till I can try it there. Worst case I can keep a conditional for el8 for now.

Reply:
An 8.5 build was successful without this. I also realized that current 8.4 package also does not have this, so I think it was introduced at some point to workaround a temporary issue in Fedora.